app/poros/gym.rb

```diff
@@ -1,6 +1,13 @@
 class Gym
-  attr_reader :yelp_gym_id, :name, :full_address, :address1, :address2, :address3,
-              :city_state_zip, :phone
+  include Convertable
+  attr_reader :yelp_gym_id,
+              :name,
+              :full_address,
+              :address1,
+              :address2,
+              :address3,
+              :city_state_zip,
```

Comment:
Perhaps we could store the address in raw format as a hash and then handle the formatting in a helper module.  But maybe I'm overthinking it.  Something to consider for future development.

Reply:
Agreed, definitely struggled a bit decision-wise with what to include and send from the BE... so I erred on the side of more vs less for now (sending both the address and raw address details from the BE, and initializing both data points within the FE PORO, so we can cherry-pick exactly what we want to extract from all that into the FE views, and not have to re-PR the BE to include additional info we could've included now.)